evm-adapters/Cargo.toml

```diff
@@ -25,7 +25,7 @@ thiserror = "1.0.29"
 proptest = "1.0.0"
 parking_lot = "0.11.2"
 futures = "0.3.17"
-revm_precompiles = { git = "https://github.com/bluealloy/revm", default-features = false, features = ["k256_ecrecover"] }
+revm_precompiles = { git = "https://github.com/bluealloy/revm", default-features = false, features = ["k256_ecrecover"], branch = "main" }
```

Comment:
Another naersk bug, it fails to resolve `main` instead of `master` by default. -_-